### PR TITLE
Change instances of staff to admin

### PR DESCRIPTION
**Pull request by MrHarshPatel**
### Tickets:

- HCK-139

### List of changes:

Changed instances of staff to admin.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Used npm run test.

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: constants/general.constant.js [hunk 1]
```diff
@@ -138,8 +138,8 @@ CREATE_ACC_EMAIL_SUBJECTS[
     VOLUNTEER
 ] = `You've been invited to create a volunteer account for ${HACKATHON_NAME}`;
 CREATE_ACC_EMAIL_SUBJECTS[
-    STAFF
-] = `You've been invited to create a staff account for ${HACKATHON_NAME}`;
+    ADMIN
+] = `You've been invited to create a admin account for ${HACKATHON_NAME}`;
```
Comment (RohitGarudadri) [inline constants/general.constant.js]: an admin*

File: package-lock.json [hunk 2]
```diff
@@ -668,7 +668,7 @@
     "@types/google-cloud__storage": {
       "version": "1.7.2",
       "resolved": "https://registry.npmjs.org/@types/google-cloud__storage/-/google-cloud__storage-1.7.2.tgz",
-      "integrity": "sha512-RaQJ7+Ht20MRYJu7mgKBpbVNZIPneztKIl/DUKacRC6A8mXRsJfgDdPA7indHmJGIgm+hzUTj44+A3RyuuYZhg==",
+      "integrity": "sha1-KFk6kMhhKZYJNE4wye5aiJWUiQE=",
```
Comment (RohitGarudadri) [inline package-lock.json]: Should these stay as sha-512??

File: docs/api/api_data.js [hunk 3]
```diff
@@ -1,3669 +1,3669 @@
 define({
-  api: [
```
Comment (pierreTklein) [inline docs/api/api_data.js]: Alright something's up here. Idk what exactly, but this change shouldn't be 7330 lines. What's the diff? Can you look into this please?

File: package-lock.json [hunk 4]
```diff
@@ -1,8605 +1,8688 @@
 {
```
Comment (pierreTklein) [inline package-lock.json]: package.json was not changed, so I don't see why package-lock.json was changed. Consider reverting this file.

**Pull request closed without merging**